spec/values/calculation/round/three_arguments.hrx

```diff
@@ -0,0 +1,247 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `round()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> positive/input.scss
+a {b: round(up, 10, 100)}
+
+<===> positive/output.css
+a {
+  b: 100;
+}
+
+<===>
+================================================================================
+<===> negative/input.scss
+a {b: round(down, -10, -100)}
+
+<===> negative/output.css
+a {
+  b: 0;
```

Comment:
Shouldn't this be -100, since we're rounding down to the next lowest multiple of -100?

Reply:
Well, the formula for the `down` strategy is: `(number.value / stepWithNumberUnit).floor() * stepWithNumberUnit` so in this case `(-10 / -100).floor() == 0` which is why it's returning a 0. For it to result in `-100`, it would have to be the `up` strategy.

Reply:
The important question isn't what the code says, it's what the spec says (ultimately this CSS spec, but in this case I think the Sass spec is correctly in agreement with it). The point of tests is to make sure the code is correct, not just to reify the code's current behavior.

Here's what the spec says:

> * Let `upper` and `lower` be the two integer multiples of `step` which
>   are closest to `number` such that `upper` is greater than `lower`. If
>   `upper` would be 0, it's specifically `-0`; if `lower` would be zero,
>   it's specifically `-0`.
> 
> * ...
>
> * If `strategy`'s value is `"down"`, return `lower`.

The lower integer multiple of -100 which is closest to -10 is -100, not 0.

Reply:
Right! With this I found a bug in the implementation. I tested the round function in Safari just to make sure, and yes `round(down, -10, -100)` should be `-100`. So it turns out that, **if the strategy is down** and the **step is negative**, we should do `.ceil()` instead of `.floor()`. Same with the `up` strategy but the other way around. I updated this in the implementation.